docs/best-practices.md

```diff
@@ -3,32 +3,8 @@ Best Practices
 
 This page is a placeholder for best practices we recommend you to review while implementing Vue InstantSearch.
 
-
-## Custom Component
```

Comment:
This was actually a move suggested by @ehayman 
The idea being to re-group all best practices on a dedicated page.